source/administration/upgrade.rst

```diff
@@ -34,6 +34,62 @@ Owner and group of the install directory - *{owner}* and *{group}*
 
 #. Review the :doc:`important-upgrade-notes` to make sure you are aware of any actions you need to take before or after upgrading from your particular version.
 
+**With a script**
+
+*Preparing the script*
+
+`Save the script<https://docs.mattermost.com/administration/update_mattermost.sh>`__ to your mattermost server.

```

Comment:
Change to this so that the link works:

_`Save the script <https://docs.mattermost.com/administration/update_mattermost.sh>`__ to your Mattermost server.